src/pages/Admin/Admin_v2.tsx

```diff
@@ -140,21 +140,20 @@ const Admin_v2: React.FC = () => {
 
   return (
     <>
-      <S.Wrapper>
-        <AdminSidebar
-          sidebarOptions={sidebarOptions}
-          selected={selectedOption}
-          onClick={setSelectedOption}
-        />
+      <S.Wrapper
+        className="no-venue-selected"
```

Comment:
`.no-venue-selected` was removed from `src/pages/Admin/Admin.styles.ts` yet is still used here. Is that correct? If so, where is the class coming from now?

Reply:
This is not being used and I'm removing it.

Reply:
Sounds good.